tests/functional/spec/answer_action_redirect_to_list_add_block_checkbox.spec.js

```diff
@@ -6,66 +6,66 @@ import AnyoneUsuallyLiveAt from "../generated_pages/answer_action_redirect_to_li
 
 describe("Answer Action: Redirect To List Add Question (Checkbox)", () => {
   describe('Given the user is on a question with a "RedirectToListAddBlock" action enabled', () => {
-    before("Launch survey", () => {
-      browser.openQuestionnaire("test_answer_action_redirect_to_list_add_block_checkbox.json");
+    before("Launch survey", async () => {
+      await browser.openQuestionnaire("test_answer_action_redirect_to_list_add_block_checkbox.json");
     });
 
-    it('When the user selects "No", Then, they should be taken to the list collector.', () => {
-      $(AnyoneUsuallyLiveAt.no()).click();
-      $(AnyoneUsuallyLiveAt.submit()).click();
-      expect(browser.getUrl()).to.contain(AnyoneLiveAtListCollector.pageName);
+    it('When the user selects "No", Then, they should be taken to the list collector.', async () => {
+      await $(AnyoneUsuallyLiveAt.no()).click();
+      await $(AnyoneUsuallyLiveAt.submit()).click();
+      await expect(await browser.getUrl()).to.contain(AnyoneLiveAtListCollector.pageName);
     });
 
-    it('When the user selects "Yes" then they should be taken to the list collector add question.', () => {
+    it('When the user selects "Yes" then they should be taken to the list collector add question.', async () => {
       browser.url(AnyoneUsuallyLiveAt.url());
-      $(AnyoneUsuallyLiveAt.iThinkSo()).click();
-      $(AnyoneUsuallyLiveAt.submit()).click();
-      expect(browser.getUrl()).to.contain(AnyoneLiveAtListCollectorAddPage.pageName);
-      expect(browser.getUrl()).to.contain("?previous=anyone-usually-live-at");
+      await $(AnyoneUsuallyLiveAt.iThinkSo()).click();
+      await $(AnyoneUsuallyLiveAt.submit()).click();
+      await expect(await browser.getUrl()).to.contain(AnyoneLiveAtListCollectorAddPage.pageName);
```

Comment:
Isn't getUrl() sync? http://v4.webdriver.io/api/property/getUrl.html

Reply:
In [this example](https://webdriver.io/docs/api/browser/url/) it uses `await` with `browser.getUrl()`.